adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -129,6 +131,7 @@ class MQTT:
     :param socket socket_pool: A pool of socket resources available for the given radio.
     :param ssl_context: SSL context for long-lived SSL connections.
     :param bool use_binary_mode: Messages are passed as bytearray instead of string to callbacks.
+    :param int socket_timeout: socket timeout, in seconds
```

Comment:
Would you mind expanding this a little, specifically what this means?  For example, I'm not very familiar with sockets so the description doesn't expand on the variable name enough that I would understand the behavior :)